code/modules/mob/mob_defines.dm

```diff
@@ -40,7 +40,7 @@
 
 	var/last_log = 0
 	var/obj/machinery/machine = null
-	var/other_mobs = null
+	var/im_in_the_middle_of_being_grab_pulled_dont_shuffle_me_pls = null
```

Comment:
I'm not one to judge and I do like me a bit of good humor but this is a very very long variable name. Your intention may be to be crystal clear in what the variable does, but perhaps a little bit of `///` documentation will be better, the previous var name was pretty vague but you could instead name this as `currently_grab_pulled` or something similair